README.md

```diff
@@ -1,2 +1,10 @@
 # HW01
-Edit the readme
+## Quick Bio of Hannah Farrell
+
+I am a rising second year PhD student in the Integrative Biology program.  
+I study human evolution, but more specifically, I am interested in the upper limb morphology of *Australopithecus afarensis*. *A. afarensis* is a **hominin**, or an evolutionary ancestor of modern humans.  
+I also love to cook and bake, here are some of my favorite things to make:  
```

Comment:
To complement this list, maybe you can insert some images of your favorite dishes.